todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskViewModel.kt

```diff
@@ -37,8 +40,9 @@ import com.example.android.architecture.blueprints.todoapp.data.source.TasksRepo
  */
 class AddEditTaskViewModel(
         context: Application,
-        private val tasksRepository: TasksRepository
-) : AndroidViewModel(context), TasksDataSource.GetTaskCallback {
+        private val tasksRepository: TasksRepository,
```

Comment:
Should we use TasksDataSource here instead of TasksRepository?

Edit: I just realised that this pull request is mostly addressing getTask, we can update this in the next pull request :)

Reply:
I think we should avoid the temptation of making any changes to their source not related to the task at hand. We may consider doing it later.